docs/building-images.md

```diff
@@ -37,6 +37,13 @@ The operating system configuration section is entirely optional.
 The following describes the possible options for the operating system section:
 ```yaml
 operatingSystem:
+  time:
+    timezone: Europe/London
+  chronyPools:
```

Comment:
My gut feeling is that the two chrony* sections should be under the umbrella `time`.

Reply:
Looking at the code, the implementation looks correct, just not this example.